include/libtorrent/torrent_info.hpp

```diff
@@ -643,7 +643,11 @@ namespace libtorrent
 			// domain in its hostname. This means the DHT and LSD
 			// features are disabled for this torrent (unless the
 			// settings allows mixing i2p peers with regular peers)
-			i2p = 4
+			i2p = 4,
+
+			// this flag is set if we found an ssl-cert field in the info
+			// dictionary
+			ssl_torrent = 8,
```

Comment:
I suspect this `,` at the end could be a problem, in that case try https://blog.travis-ci.com for auto cancelation of PRs (new)